src/installer/__main__.py

```diff
@@ -0,0 +1,206 @@
+"""Installer CLI."""
+
+import argparse
+import compileall
+import distutils.dist
+import os
+import os.path
+import platform
+import sys
+import sysconfig
+import warnings
+
+import installer
+import installer.destinations
+import installer.sources
+import installer.utils
+from installer._compat.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from email.message import Message
+    from typing import Any, Dict, List, Optional, Sequence, Union
+
+    from installer._compat.typing import Text
+    from installer.records import RecordEntry
+
+
+def main_parser():  # type: () -> argparse.ArgumentParser
+    """Construct the main parser."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("wheel", type=str, help="wheel file to install")
+    parser.add_argument(
+        "--destdir",
+        "-d",
+        metavar="/",
+        type=str,
+        default="/",
+        help="destination directory",
```

Comment:
That is indeed what it does -- the name "destdir" comes from the fact that it's traditionally called "destdir" rather than "root" in most packaging ecosystems (as popularized by [the GNU coding standards](https://www.gnu.org/prep/standards/html_node/DESTDIR.html)), including the CPython Makefile.

e.g.

```
DESTDIR="/path/to/staging/root-filesystem" make install
meson install --destdir "/path/to/staging/root-filesystem"
```